src/components/TradeAssetSearch/TradeAssetSearch.tsx

```diff
@@ -162,12 +162,10 @@ export const TradeAssetSearch: FC<TradeAssetSearchProps> = ({
       return walletConnectedChainIds
     })()
 
-    const sortedChainIds = sortChainIdsByDisplayName(unsortedChainIds).filter(chainId =>
-      isSwapper ? chainId !== KnownChainIds.SolanaMainnet : true,
```

Comment:
q: did we remove this because we have the security at asset filtering step?

Suggesting we disable the Solana guard on line 149 and 109 if the feature flag is on, wdyt?

Reply:
Yeah as discussed this was a monkey-patch to hide Solana assets from all search bits, which we should've reverted or at the very least tied to Solana flag (which is now on in all envs)

Reply:
38492b010512e005b54d95c207d3c6e6fef2a00